src/components/sync-connected-sites.tsx

```diff
@@ -362,10 +367,17 @@ const SyncConnectedSiteSection = ( {
 		isSiteIdPushing( selectedSite.id, site.id )
 	);
 
+	let logo = <WordPressLogoCircle />;
+	if ( hasConnectionErrors ) {
+		logo = <CircleRedCrossIcon />;
+	} else if ( mainSite?.isPressable ) {
+		logo = <PressableLogo />;
+	}
```

Comment:
Nice!

You could also create a variable that starts with capital letter

```
	let Logo = WordPressLogoCircle;
	if ( hasConnectionErrors ) {
		Logo = CircleRedCrossIcon;
	} else if ( mainSite?.isPressable ) {
		Logo = PressableLogo;
	}
```

and then use it as a regular component:
`<Logo />`

Reply:
That's cleaner! Thanks for sharing, Antonio!